app/frontend/src/service/__mocks__/index.ts

```diff
@@ -1,101 +0,0 @@
-import { service as originalService } from "..";
```

Comment:
Deleted this as this was also causing a lot of headache when trying to debug, since console logging a proxy gives you back a Symbol object...

Jest mocking correctly mocks out everything deeply anyway when we do `jest.mock` on this, so this is unnecessary code and it's interfering with the good default. And since the implementations in each mocks get reset, this is only good for one test anyway so it's useless. I've moved the default implementations into its own file in the `test` folder where all the test helpers are instead.

Reply:
Sure. How are most of the tests still passing without changes? Because they didn't really use this?

Reply:
Yeah this wasn't being used, the existing tests either set the mock implementation before each test or had to manually override the default mock implementation to trigger certain scenarios (e.g. throwing an error).